tests/foundry/test/AddrTest.t.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity =0.8.13;
+
+import "forge-std/Test.sol";
+
+contract AddrTest is Test {
+
+    function testAddr() public {
+        address alice = vm.addr(1);
+        assertEq(alice, 0x7E5F4552091A69125d5DfCb7b8C2659029395Bdf);
```

Comment:
What about having two tests, one with `assert` as you suggest to test out the cheat code itself, and another with `assertEq` to test the integration? I think using `assertEq` may be more idiomatic than `assert` when using Foundry, so maybe users do this more often. But maybe integration with other cheat codes is a pandora's box we don't want to open yet

Reply:
Yeah, I agree that in general `assertEq` should be used. A compromise could be to only test the `load`/`store` cheatcodes with `assert` to make sure they work. Once they pass, we can use `assertEq` for all other cheatcodes. So maybe we should just really quickly add support for `load`/`store` so that we can use `assertEq`.

Reply:
I left this tests and added 4 more tests using `assert`. Two of them are failing tests (similar to `AssumeTest.t.sol`).